Test/ServerTelemetryChannel.Test/Shared.Tests/SamplingTelemetryProcessorTest.cs

```diff
@@ -374,34 +385,73 @@ private static void TelemetryTypeDoesNotSupportSampling(Func<TelemetryProcessorC
             Assert.Equal(generatedCount, sentTelemetry.Count);
         }
 
-        private static void TelemetryTypeSupportsSampling(Action<TelemetryProcessorChain> sendAction, string excludedTypes = null, string includedTypes = null)
+        private static void TelemetryTypeSupportsSampling(Action<TelemetryProcessorChain> sendAction,
+                                                          string excludedTypes = null,
+                                                          string includedTypes = null)
+        {
+            TelemetryTypeSupportsSampling((chain) => { sendAction(chain); return 1; },
+                                          excludedTypes,
+                                          includedTypes);
+        }
+
+        private static void TelemetryTypeSupportsSampling(Func<TelemetryProcessorChain, int> sendAction,
+                                                          string excludedTypes = null,
+                                                          string includedTypes = null,
+                                                          int samplingPercentage = 10)
         {
             const int ItemsToGenerate = 100;
-            const int SamplingPercentage = 10;
             var sentTelemetry = new List<ITelemetry>();
-            var telemetryProcessorChainWithSampling = CreateTelemetryProcessorChainWithSampling(sentTelemetry, SamplingPercentage, excludedTypes, includedTypes);
+            TelemetryProcessorChain telemetryProcessorChainWithSampling = CreateTelemetryProcessorChainWithSampling(sentTelemetry,
+                                                                                                                    samplingPercentage,
+                                                                                                                    excludedTypes,
+                                                                                                                    includedTypes);
```

Comment:
I tried to stick to existing formatting everywhere and StyleCop runs clean. I could have missed something, so please let me know if you would like me to adjust something specific. On the first glance, this looks fine to me.